coalib/parsing/Globbing.py

```diff
@@ -299,33 +306,35 @@ def relative_wildcard_glob(dirname, pattern):
     return result
 
 
-def relative_flat_glob(dirname, basename):
+def relative_flat_glob(dirname, basename, ignored_globs=None):
```

Comment:
yes, but in globbing.py in _iglob function, there is a variable names relative_glob_function which can get this function assigned. And that relative_glob_function is called with 2 parameters.